src/firecfg/firecfg.config

```diff
@@ -554,6 +554,7 @@ microsoft-edge-beta
 microsoft-edge-dev
 microsoft-edge-stable
 midori
+mimetype
```

Comment:
```suggestion
```

Same here.

It seems to contain both a general-purpose user-facing tool and a library, both
of which are intended to be used on arbitrary files, so enabling it by default
seems risky.

Also, it seems like it could potentially affect a lot of unrelated programs:

```
$ LC_ALL=C pacman -Sii extra/perl-file-mimeinfo |
  grep -e '^Required By' -e '^Optional For'
Required By     : auto-multiple-choice
Optional For    : xdg-utils
$ LC_ALL=C pacman -Sii extra/xdg-utils
Repository      : extra
Name            : xdg-utils
Version         : 1.2.1-1
Description     : Command line tools that assist applications with a variety of desktop integration tasks
[...]
Required By     : ario  blender  catfish  chromium  cuyo  deepin-daemon  deepin-printer  deluge-gtk  fff  filezilla  flatpak
                  gambas3-runtime  glob2  gramps  hplip  httrack  jre8-openjdk  keepass  libreoffice-fresh  libreoffice-still  linssid
                  minetest  mumble  nextcloud-client  openconnect  openra  qmmp  qt5-base  qt6-base  retext  rmlint-shredder
                  rofi-pass  shutter  subtitleeditor  surge-xt-clap  surge-xt-standalone  surge-xt-vst3  sxiv  synapse  terminator
                  wire-desktop  wireshark-qt  wiznote  xboard  xpdf
Optional For    : arch-wiki-lite  cliphist  cups  discord  electron22  electron23  electron27  electron28  electron29  foot  gscan2pdf
                  intel-oneapi-basekit  nnn  weechat-matrix  wl-clipboard  xarchiver  xarchiver-gtk2
```

Reply:
Removed it from firecfg.